CHANGELOG.md

```diff
@@ -326,6 +326,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Disable loading dataloades if corresponding `limit_batches=0` ([#11576](https://github.com/PyTorchLightning/pytorch-lightning/pull/11576))
 
 
+- Only allow one value for each type in `plugins` trainer flag ([#12083](https://github.com/PyTorchLightning/pytorch-lightning/pull/12083))
```

Comment:
does this need a changelog entry since this was already available before?

Reply:
I think this is new, it wasn't checked before

Reply:
This was checked in 1.5.10 and raised an error. 
I verified with

```py
    trainer = Trainer(
        accelerator="cpu",
        devices=2,
        strategy="ddp",
        plugins=[DDPSpawnPlugin()]
    )
```

```py
    trainer = Trainer(
        accelerator="cpu",
        devices=2,
        plugins=[DDPSpawnPlugin(), DDPPlugin()]
    )
```

```py
    trainer = Trainer(
        accelerator="cpu",
        devices=2,
         plugins=[PrecisionPlugin(), NativeMixedPrecisionPlugin(16, device="cpu")]

```


So as far as I can see, this is fixing a AcceleratorConnector bug on master.

Reply:
So I recommend to remove the changelog entry.